pkg/shp/cmd/buildrun/logs.go

```diff
@@ -85,7 +86,8 @@ func (c *LogsCommand) Run(params *params.Params, ioStreams *genericclioptions.IO
 	// is invoked.
 	justGetLogs := false
 	var pods *corev1.PodList
-	err = wait.PollImmediate(1*time.Second, 10*time.Second, func() (done bool, err error) {
+	ctx := context.Background()
+	err = wait.PollUntilContextTimeout(ctx, 1*time.Second, 10*time.Second, true, func(context.Context) (done bool, err error) {
```

Comment:
Curious, couldn't we use `c.cmd.Context()` instead of a new background context or the `PollUntilContextTimeout` call?

Reply:
Fixed in next push